go/ct/gen/state.go

```diff
@@ -372,7 +402,52 @@ func (g *StateGenerator) Generate(rnd *rand.Rand) (*st.State, error) {
 		return nil, fmt.Errorf("failed to resolve gas refund constraints: %w", err)
 	}
 
-	accountAddress := RandomAddress(rnd)
+	// --- Self Address ---
```

Comment:
nit: this could be move into its own function to reduce the already large size of `Generate`

Reply:
I do not want to break the style for one function here now, mixing styles. This function should be refactored as a whole eventually.